src/app/App.module.css

```diff
@@ -0,0 +1,41 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200;300;400&display=swap');
```

Comment:
You could import the font in global.css and set the font-family for the whole project to 'Source Sans'